src/Umbraco.Cms.Persistence.EFCore/Constants-ProviderNames.cs

```diff
@@ -0,0 +1,11 @@
+namespace Umbraco.Cms.Persistence.EFCore;
+
+public static partial class Constants
+{
+    public static class ProviderNames
+    {
+        public const string SQLLite = "Microsoft.Data.Sqlite";
+
+        public const string SQLServer = "Microsoft.Data.SqlClient";
+    }
+}
```

Comment:
@Zeegaan would it be better to move the constant to "Umbraco.Core" project, where we have most constants?
https://github.com/nikcio/Umbraco-CMS/tree/1fa09f37fc310c1b14bf68aac2394d592631c8c9/src/Umbraco.Core

Then it can be used without reference to `Umbraco.Cms.Persistence.EFCore`.

Reply:
It can yes, good point 👍